### PR TITLE
Change --reads to --input everywhere

### PR DESCRIPTION
**Pull request by drpatelh**

### CODE REVIEW

File: public_html/index.php [hunk 1]
```diff
@@ -177,9 +177,9 @@
 
 <span class="text-white-50"># Launch the RNAseq pipeline</span>
 nextflow run nf-core/rnaseq \
-    -profile docker \
+    --input samplesheet.csv \
```
Comment (mashehu) [inline public_html/index.php:180]: Is that change already released in the rnaseq pipeline?

Comment (drpatelh) [reply public_html/index.php:180]: Good point 👍 Not yet but imminent (he says). Can leave this open if you like?

**Review by ewels — APPROVED** [review]
LGTM but let's wait for the rnaseq release 👍🏻

**Comment by drpatelh** [comment]
Thanks @mashehu !

**Pull request merged**